test.py

```diff
@@ -0,0 +1,7 @@
+import characters
+
+check_character_input = input("Which roll do you want to use? ")
+num_dice_input = characters.weapons.get(check_character_input)[0]
+modifier = characters.weapons.get(check_character_input)[1]
+print(num_dice_input)
+print(modifier)
```

Comment:
this is a great start to having tests! For a test you could use a static value for inputs instead of asking the test user for something and then make use of the `assert` method.